controller/controller_test.go

```diff
@@ -470,3 +530,166 @@ func TestController_Status_ErrorOnMismatchingSliceIDs(t *testing.T) {
 	Expect(status).To(BeEmpty())
 	mock.AssertExpectationsForObjects(t, fleetMock.Mock)
 }
+
+func TestController_WaitForStatus_001(t *testing.T) {
```

Comment:
Something that gives me a hint what this test wants to prove. At first glance it seems to simulate a unit switching from activating to running, so maybe `TestController_WaitForStatus_UnitActivatingToRunning` ?